cmd/kind/app/main_test.go

```diff
@@ -78,3 +80,59 @@ func TestCheckQuiet(t *testing.T) {
 		})
 	}
 }
+
+func Test_CommandErrReturn(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		Name       string
+		Command    string
+		Subcommand string
```

Comment:
should we add a `expectErr` field and do some positive testing? i..e commands that we know can't fail

Reply:
only if we turn this into an integration test, which can be in a follow-up (let's not be running real commands in unit tests)